applications/tari_app_utilities/src/identity_management.rs

```diff
@@ -49,7 +49,7 @@ pub fn setup_node_identity<P: AsRef<Path>>(
     peer_features: PeerFeatures,
 ) -> Result<Arc<NodeIdentity>, ExitCodes>
 {
-    match load_identity(identity_file.as_ref().clone()) {
+    match load_identity(<&std::path::Path>::clone(&identity_file.as_ref())) {
```

Comment:
`load_identity` takes a `P: AsRef<Path>` which is what `identity_file` already is, so this will work:

```suggestion
    match load_identity(identity_file) {
```

Reply:
Needed a reference, as the value is borrowed further down